rita/src/rita_client/dashboard/neighbors.rs

```diff
@@ -1,11 +1,12 @@
 use super::*;
+use num256::Uint256;
 
 #[derive(Serialize)]
 pub struct NodeInfo {
     pub nickname: String,
     pub route_metric_to_exit: u16,
-    pub total_payments: Int256,
-    pub debt: i128,
+    pub total_payments: Uint256,
+    pub debt: Int256,
```

Comment:
is this properly displayed such that it's not going to have hex values?

Reply:
Good idea. No reason to keep current behavior. I'll address it here: althea-mesh/num256_rs#10

Reply:
Solved in num256_rs 0.2.4